### PR TITLE
Updated Action to use Nuitka 1.3.8 and added options for show-scons and building with MinGW64

### PR DESCRIPTION
**Pull request by WhosMyName**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -97,6 +101,11 @@ inputs:
     description: Description of the file use in Windows Version information.  One of file or product version is required, when a version resource needs to be added, e.g. to specify product name, or company name. Defaults to nonsense.
   windows-onefile-tempdir-spec:
     description: "Use this as a temporary folder. Defaults to '%TEMP%\\onefile_%PID%_%TIME%', i.e. system temporary directory."
+  windows-mingw-build:
```
Comment (kayhayen) [inline action.yml:104]: I would love for us to keep using the option names, in this case `ming64` then.

**Comment by kayhayen** [comment]
@jimkring outside of the option name for `--mingw64`, I do not have complaints, I suggest you merge this one?

One thing I noticed however, is that the default Nuitka version is not just a branch name, like `main`, can we do it like that, I don't think it's good to update this for every hotfix or Nuitka.

**Review by kayhayen — CHANGES_REQUESTED** [review]
If you change the Yaml option name to match the command line name, it will be perfect. We intend to actually read the command line options from the Yaml file in the future, then it would be great if they already matched as much as possible.

**Comment by WhosMyName** [comment]
change implemented in bc4a461
ready to merge

**Comment by kayhayen** [comment]
Merged this. After looking at it, I am somewhat more in doubt, if Nuitka options can be done automatically from the Yaml configuration. It seems we might need some sort of code generator to achieve that. Not sure if it is worth it.

**Pull request merged**